From d9cb3aeb75ec1881084eb1d5958f9cbfd08d9e83 Mon Sep 17 00:00:00 2001
From: "wei.nie" <wei.nie@amlogic.com>
Date: Thu, 10 Jul 2014 17:18:54 +0800
Subject: [PATCH 4500/5965] PD#92307:m8baby:HDMI:enable HDMI dual out for
 tablet project.

zhenfei.li and niewei fix error for followong commit:
http://10.8.9.5/kernel/common/commit/arch/arm/mach-meson8b/include/mach/power_gate.h?h=amlogic-3.10-bringup&id=4c08fa7a7835a541d4f1fdc75489714ab2c756f6
---
 arch/arm/mach-meson8b/include/mach/power_gate.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-meson8b/include/mach/power_gate.h b/arch/arm/mach-meson8b/include/mach/power_gate.h
index ac7bb760dc32..946b86f20f75 100755
--- a/arch/arm/mach-meson8b/include/mach/power_gate.h
+++ b/arch/arm/mach-meson8b/include/mach/power_gate.h
@@ -17,7 +17,7 @@
 
 #define __CLK_GATE_OFF(_MOD) \
 	do{                             \
-		if(GCLK_ref[GCLK_IDX_##_MOD] <= 0){ \
+		if(--GCLK_ref[GCLK_IDX_##_MOD] <= 0){ \
 			if (0) printk(KERN_INFO "gate off %s %x, %x\n", GCLK_NAME_##_MOD, GCLK_REG_##_MOD, GCLK_MASK_##_MOD); \
 			CLEAR_CBUS_REG_MASK(GCLK_REG_##_MOD, GCLK_MASK_##_MOD); \
 			GCLK_ref[GCLK_IDX_##_MOD] = 0; \
-- 
2.19.0

